### PR TITLE
Cleanup Choco Install Process

### PR DESCRIPTION
**Pull request by isaacrlevin**
- Delete unneeded file
- Add settings backup step to upgrade

### CODE REVIEW

**Comment by isaacrlevin** [comment]
@gep13 is this going to work? ;)

**Comment by gep13** [comment]
@isaacrlevin I have added a comment here https://github.com/isaacrlevin/presencelight/issues/515#issuecomment-998672087 to talk about this, as I don't think that some of the work here is needed.

File: .chocolatey/tools/ChocolateyBeforeModify.ps1 [hunk 1]
```diff
@@ -0,0 +1,11 @@
+$InstallDir = Join-Path $env:APPDATA 'PresenceLight'
```
Comment (gep13) [inline .chocolatey/tools/ChocolateyBeforeModify.ps1:1]: If you move forward with this, this doesn't take into account the ability to configure that installation directory via a Package Parameter, which is done here:

https://github.com/isaacrlevin/presencelight/blob/main/.chocolatey/tools/ChocolateyInstall.ps1#L41

File: .chocolatey/tools/ChocolateyBeforeModify.ps1 [hunk 2]
```diff
@@ -0,0 +1,11 @@
+$InstallDir = Join-Path $env:APPDATA 'PresenceLight'
+$SettingsPath = Join-Path $InstallDir 'appsettings.json'
```
Comment (gep13) [inline .chocolatey/tools/ChocolateyBeforeModify.ps1:2]: Based on the testing that I did, the custom information was being added to a `settings.json` file, not the `appsettings.json` file.

File: .chocolatey/tools/ChocolateyInstall.ps1 [hunk 3]
```diff
@@ -51,6 +51,18 @@ $packageArgs = @{
...
 
+$TempSettingsPath = Join-Path $env:TEMP 'PresenceLightSettings.json'
+
+if (Test-Path -Path $TempSettingsPath -PathType Leaf)
+{
+    $SettingsPath Join-Path $InstallDir 'appsettings.json'
+    Remove-Item -Path $SettingsPath -Recurse
+    
+    Get-ChildItem -path $TempSettingsPath | 
+    Copy-Item -Destination $InstallDir -PassThru | Rename-Item -NewName  'appsettings.json'
```
Comment (gep13) [inline .chocolatey/tools/ChocolateyInstall.ps1:62]: Same comment here, it was the `settings.json` file that stored the custom settings from the UI.  At least in the test that I did.

**Review by gep13 — CHANGES_REQUESTED** [review]

**Comment by isaacrlevin** [comment]
Closing as it seems this is not needed

**Pull request closed without merging**